client/src/services/ApiClient.js

```diff
@@ -90,6 +90,23 @@ export async function addFundToWallet(clientID, increaseBy) {
   }
 }
 
+export async function createClient(client) {
+ 
+  const response = await fetch("http://localhost:3000/api/clients", {
+    method : 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify({...client, fullName: client.firstName +" "+ client.lastName,
+     address: client.address +", "+ client.number+" "+ client.city+ ", "+client.postCode, wallet: 0})
+  });
+  
+
+  if(response.OK){
```

Comment:
Follow the switch template as we did for other fetch requests to handle negative responses